dpnp/dpnp_utils/dpnp_utils_einsum.py

```diff
@@ -853,6 +800,12 @@ def _parse_ellipsis_subscript(subscript, idx, ndim=None, ellipsis_len=None):
 def _parse_int_subscript(list_subscript):
     """Copied from _parse_int_subscript in cupy/core/_einsum.py"""
 
+    if not isinstance(
+        list_subscript, (list, tuple, numpy.ndarray, dpnp.ndarray)
```

Comment:
Do we want to work with individual array here or do we want to work with the list?
Please change the input argument check or error description.